Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -844,30 +1090,361 @@ def lift_host_containment_command():
     return create_entry_object(contents=raw_res, hr=hr)
 
 
+def run_command():
+    args = demisto.args()
+    host_ids = argToList(args.get('host_ids'))
+    command_type = args.get('command_type')
+    full_command = args.get('full_command')
+    scope = args.get('scope', 'read')
+
+    if scope == 'read':
+
+        response = run_batch_read_cmd(host_ids, command_type, full_command)
+    elif scope == 'write':
+        response = run_batch_write_cmd(host_ids, command_type, full_command)
+    else:  # scope = admin
+        response = run_batch_admin_cmd(host_ids, command_type, full_command)
+
+    resources: dict = response.get('combined', {}).get('resources', {})
+
+    output = []
+
+    for _, resource in resources.items():
+        errors = resource.get('errors', [])
+        if errors:
+            error_message = errors[0].get('message', '')
+            if not error_message:
+                error_message = f'Could not run command\n{errors}'
+            return_error(error_message)
+        output.append({
+            'HostID': resource.get('aid'),
+            'Stdout': resource.get('stdout'),
+            'Stderr': resource.get('stderr'),
+            'BaseCommand': resource.get('base_command'),
+            'Command': full_command
+        })
+
+    human_readable = tableToMarkdown(f'Command {full_command} results', output, removeNull=True)
+    entry_context = {
+        'CrowdStrike': {
+            'Command': output
+        }
+    }
+
+    return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def upload_script_command():
+    args = demisto.args()
+    name = args.get('name')
+    permission_type = args.get('permission_type', 'private')
+    content = args.get('content')
+    entry_id = args.get('entry_id')
+
+    if content and entry_id:
+        raise ValueError('Only one of the arguments entry_id or content should be provided, not both.')
+    elif not content and not entry_id:
+        raise ValueError('One of the arguments entry_id or content must be provided, none given.')
+
+    response = upload_script(name, permission_type, content, entry_id)
+
+    return create_entry_object(contents=response, hr='The script was uploaded successfully')
+
+
+def get_script_command():
+    script_id = argToList(demisto.args().get('script_id'))
+
+    response = get_script(script_id)
+
+    resources: list = response.get('resources', [])
+    if resources and isinstance(resources, list):
+        resource = resources[0]
```

Comment:
what about the rest of the list? if it's only one, add a comment to specify that